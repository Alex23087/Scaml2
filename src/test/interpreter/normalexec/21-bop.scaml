let x = 10 + 20
let y = x - 25
let z = y * 33
let w = z / 11

let _ =
  print x;
  print "\n";
  print y;
  print "\n";
  print z;
  print "\n";
  print w;
  print "\n";
  []