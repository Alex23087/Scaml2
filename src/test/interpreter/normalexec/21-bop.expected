30
5
165
15
